dist/pom.xml

```diff
@@ -113,6 +113,17 @@
       <artifactId>zeebe-cluster-topology</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>operate-webapp</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>commons-logging</groupId>
+          <artifactId>commons-logging</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
```

Comment:
💭 Just want to raise, that this means from now on we have to build Operate and their frontend if we run Zeebe integration tests. @npepinpe mentioned to me last week that QA depends on dist. We might want to change that at some point again, maybe introducing separate dist modules in the subfolders, such that we just have on the root a dist folder that references the others.

Reply:
I expect the main impact to be for when you want to run our QA tests locally, but this is just an assumption yet :shrug: